jinja2cli/cli.py

```diff
@@ -178,6 +178,8 @@ def cli(opts, args):
 
     output = render(template_path, data, extensions)
 
+    if isinstance(output, bytes):
```

Comment:
Wouldn't we also want to do this for `str` for python2?

Can I propose something like:

```python
PY3 = sys.version_info[0] == 3

if PY3:
  binary_type = bytes
else:
  binary_type = str

# ...

if isinstance(output, binary_type):
  output = output.decode('utf-8')
```